wallet/bindings/java/lib/native/src/lib.rs

```diff
@@ -49,7 +49,7 @@ macro_rules! jni_err_assert {
 // it throws an exception and returns the value of r. Otherwise returns a String.
 macro_rules! string_from_jni {
     ($env:ident, $x:ident, $r:expr ) => {{
-        let string = $env.get_string($x);
+        let string = $env.get_string_unchecked(&$x);
```

Comment:
Is this alright? I don't remember having to do this when I updated JNI in the client 🤔

Reply:
yea, its slightly faster then the get_string version as it.. doesnt check the type is a java.lang.String. We check all this on java side